sdks/python/apache_beam/ml/inference/pytorch_inference_test.py

```diff
@@ -609,6 +609,57 @@ def test_gpu_auto_convert_to_cpu(self):
           "are not available. Switching to CPU.",
           log.output)
 
+  def test_load_torch_script_model(self):
+    torch_model = PytorchLinearRegression(2, 1)
+    torch_script_model = torch.jit.script(torch_model)
+
+    torch_script_path = os.path.join(self.tmpdir, 'torch_script_model.pt')
+
+    torch.jit.save(torch_script_model, torch_script_path)
```

Comment:
Does this work for remote models? If so, can we just use a remote model in gcs rather than saving a model for each test?

Reply:
These are pretty light models and takes ~3 seconds to run. 

Also, the disadvantage of saving the models in GCS bucket for unittests is that we may need to manually update them when the pytorch future version becomes incompatible in any case.

Reply:
Ok, leaving it works for me.